README.md

```diff
@@ -8,6 +8,18 @@ The default configuration will:
 - Close the stale issues and pull requests after 7 days of inactivity
 - If an update/comment occur on stale issues or pull requests, the stale label will be removed and the timer will restart
 
+## Recommended Permissions
+
+For the execution of the action, it must be able to read all issues and pull requests. In addition, the action must have the permission to write to issues and/or pull requests. This can be achieved with the following [configuration in the action](https://docs.github.com/en/actions/reference/workflow-syntax-for-github-actions#permissions) if the permissions are restricted:
+
+```yml
+permissions:
+  issues: write
+  pull-requests: write
+```
+
+You can find more detailed information about the required permissions under the corresponding options.
```

Comment:
```suggestion
For the execution of this action, it must be able to fetch all issues and pull requests from your repository.  
This can be achieved with the following [configuration in the action](https://docs.github.com/en/actions/reference/workflow-syntax-for-github-actions#permissions) if the permissions are restricted:

yml ⚠️ >>> wrap this as code it's just an issue with the suggestion format
permissions:
  issues: write
  pull-requests: write


In addition, based on the provided configuration, the action could require more permission(s) (e.g.: add label, remove label, comment, close, etc).  
You can find more information about the required permissions under the corresponding options that you wish to use.
```

Reply:
Thanks for the suggestion! 😄